test/sql/test_function/T/test_cast

```diff
@@ -0,0 +1,23 @@
+-- name: test_cast_string_to_varbinary
+
+create table t1(c1 int, c2 string);
+create table t2(c1 int, c2 binary);
+create table t3(c1 int, c2 varbinary);
+
+insert into t1 values (1, "1,2,3,4,5");
+insert into t1 values (2, null);
+insert into t2 select * from t1;
+insert into t3 select * from t1;
+
+select * from t2 order by c1;
+select * from t3 order by c1;
+
+-- name: test_cast_varbinary_to_string
+create table t1(c1 int, c2 binary, c3 varbinary);
+create table t2(c1 int, c2 string, c3 string);
+
+insert into t1 values (1, "1,2,3,4,5", "6,7,8,9");
+insert into t1 values (2, null, null);
+insert into t2 select * from t1;
+
+select * from t2 order by c1;
```

Comment:
you can add some tests of explicit cast functions

Reply:
Fixed